### PR TITLE
 Simplified Installation: one line installation 

### PR DESCRIPTION
**Pull request by NazarKY**
We should ask users minimum changes in the code. As for me for rails app will fine to have:

```
gem 'oauth2me'
```

will use default scopes and for each omniauth provider setup default api keys.

### CODE REVIEW

**Comment by NazarKY** [comment]
@tmwh: pls review

File: lib/oauth2-me.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 require 'base64'
 require 'oauth2-me/omniauth_strategy_updater'
+require 'oauth2-me/railtie'
```
Comment (tmwh) [inline lib/oauth2-me.rb]:
```
if defined?(Rails)
```

**Comment by NazarKY** [comment]
@tmwh pls review

**Pull request merged**